src/main/java/resourses/input.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
This folder should be in the main folder 
<img width="232" alt="image" src="https://user-images.githubusercontent.com/68428976/161762152-ca03c7cd-014d-41fb-9747-7931c0d4098a.png">